src/openzaak/components/catalogi/tests/test_resultaattype.py

```diff
@@ -438,23 +428,18 @@ def test_derive_archiefactiedatum_from_selectielijstklasse(self, *mocks):
             },
         }
 
-        responses = {
-            SELECTIELIJSTKLASSE_URL: {
-                "url": SELECTIELIJSTKLASSE_URL,
-                "procesType": PROCESTYPE_URL,
-                "procestermijn": Procestermijn.nihil,
-            }
-        }
+        with requests_mock.Mocker() as m:
```

Comment:
Is there a preferred way to use `requests_mock.Mocker`, decorator vs context manager?

Reply:
not really - in a number of cases I went with it because there are a number of `unittest.mock.patch` calls which are passed as `*mocks`, and the requests mock stuff is injected _after_ those as an `arg` rather than `kwargs` so that makes it annoying in terms of method signature. using the context manager is then simpler to not have to mess around with that.

I think I personally prefer the decorator because the context manager introduces a new level of nesting and I prefer code to be "as flat as possible".